src/MLJInterface.jl

```diff
@@ -137,12 +188,28 @@ function MMI.fit(m::AbstractSRRegressor, verbosity, X, y, w=nothing)
     return MMI.update(m, verbosity, nothing, nothing, X, y, w)
 end
 function MMI.update(
-    m::AbstractSRRegressor, verbosity, old_fitresult, old_cache, X, y, w=nothing
+    m::AbstractSRRegressor,
+    verbosity,
+    old_fitresult::Union{SRFitResult,Nothing},
```

Comment:
(Maybe it's not getting passed as an `SRFitResult` here? Can just try using `Any` to see if it fixes things)